README.md

```diff
@@ -27,6 +28,33 @@ The SDK is a Kotlin library for sending user properties and events to the Appcue
 
 ## 🚀 Getting Started
 
+### Prerequisites
+
+Your application's `build.gradle` must have a `compileSdkVersion` of 34+ and `minSdkVersion` of 21+
+
+```
+android {
+    compileSdkVersion 34
+
+    defaultConfig {
+        minSdkVersion 21
+    }
+}
+```
+
+In addition because of Compose 1.6 update (internally used by the SDK), it is required to either:
```

Comment:
could say: 
Due to the SDK usage of Jetpack Compose, it is required to either:

that way it is evergreen for future versions until we need to change?